pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -808,18 +808,24 @@ not provided, it defaults to @racket[equal?].
  A @racket[match] pattern form that matches an instance of a structure
  type named @racket[struct-id], where the field @racket[field] in the
  instance matches the corresponding @racket[pat].
+ The fields do not include those from super types.
```

Comment:
Nope. It has been like this (perhaps unintentionally, but its behavior makes sense because both child and parent structs could have the same field name). This PR makes it clear what the behavior is.